Docs/ZynqVivadoXsa.md

```diff
@@ -0,0 +1,107 @@
+# Building Vivado XSA (Xilinx Shell Archive) file for Hastlayer (Trenz TE0715-04-30-1C module)
+
+To start with the Hastlayer on the Trenz TE0715-04-30-1C module you first need to build a Vivado XSA file containing the appropriate Zynq Processing System (ARM CPU) and FPGA fabric configuration. This document describes the required steps.
+
+## Install Vitis Core Development Kit 2020.2
+
+Download the Vitis Core Development Kit 2020.2 installer (Vivado included) from Xilinx website: (https://www.xilinx.com/support/download/index.html/content/xilinx/en/downloadNav/vitis/2020-2.html) and install it. 
+
+![Download Vitis Core Development Kit Installer](Images/VitisDownloadInstaller.png)
+
+You can install Vitis and Vivado as root user in the default /tools/Xilinx folder.
+
+Before using the Vivado tools you first have to set up the environment:
+
+```
+source /tools/Xilinx/Vitis/2020.2/settings64.sh
+```
+
+## Downloading and configuring the reference design for TE0715-04-30-1C module
+
+Create an empty directory, then download and extract the reference design from Trenz Electronic:
+
+```
+mkdir ${HOME}/trenz_te0715_04_30_1c
+cd ${HOME}/trenz_te0715_04_30_1c
+mkdir vivado
+cd vivado
+wget https://shop.trenz-electronic.de/trenzdownloads/Trenz_Electronic/Modules_and_Module_Carriers/4x5/TE0715/Reference_Design/2020.2/test_board/TE0715-test_board-vivado_2020.2-build_5_20210611100936.zip
```

Comment:
Ez a link halott. Úgy látom amikor frissítik (legutoljára december 16-án) akkor a korábbit letörlik ezért nem lesz jó így hotlinkelni.
Helyette le kéne írni, hogy innen kell letölteni:
https://shop.trenz-electronic.de/Download/?path=Trenz_Electronic/Modules_and_Module_Carriers/4x5/TE0715/Reference_Design/2020.2

Reply:
Erre is válaszolj majd, és kérlek ne nyomj a <kbd>Resolve conversation</kbd>-re. Az a reviewer feladata. (Persze ha nyomsz <kbd>Commit suggestion</kbd>-t akkor a rendszer magátol resolve-olja, az rendben van.)